Classes/Notifications/NewFeaturesController.swift

```diff
@@ -64,17 +64,25 @@ final class NewFeaturesController {
         guard let url = fetchURL,
             (testing == true || hasFetchedLatest == false)
             else { return }
-        hasFetchedLatest = true
 
-        let task = session.dataTask(with: url) { (data, response, _) in
-            if let response = response as? HTTPURLResponse,
-                response.statusCode == 200,
-                let data = data,
-                let string = String(data: data, encoding: .utf8) {
-                DispatchQueue.main.async { [weak self] in
-                    self?.latestMarkdown = string
-                    success()
-                }
+        let task = session.dataTask(with: url) { data, response, error in
+            guard
+                let httpResponse = response as? HTTPURLResponse,
+                [200, 404].contains(httpResponse.statusCode),
```

Comment:
I am not sure if we need to take into consideration `404`. If the status is `404` and rest of the guard passed, means that `hasFetchedLatest` will be `true`. Which, in IMO, shouldn't be. 🤔

Reply:
I think we need to care only about this range `200..<300`.